# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/muffin/lower-udev.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- muffin-6.2.0/meson.build.vanilla	2024-09-06 23:46:43.699313169 +0200
+++ muffin-6.2.0/meson.build	2024-09-06 23:47:14.821567401 +0200
@@ -33,7 +33,7 @@
 atk_req = '>= 2.5.3'
 
 # optional version requirements
-udev_req = '>= 228'
+udev_req = '>= 182'
 gudev_req = '>= 232'
 
 # wayland version requirements
